### ISSUE TITLE
Converting a base checkpoint to an MVDream checkpoint

### ISSUE THREAD

**Issue by KappaMund** [bytedance/MVDream#4]
Is it possible to convert other checkpoints to MVDream checkpoint, for example basic SDXL checkpoint or custom SDXL 2.1?

**Comment by seasonSH** [comment]
No, we fine-tune the model to learn multi-view consistency, so you will need to do the same for other models such as SDXL. For other SD2.1 models (e.g. LORA), you may try to merge the models, not sure about consistency after that though...

**Comment by andriilitvynchuk** [comment]
@seasonSH is it possible to get original SD2.1 weights (I mean before your fine-tuning)? As I understand from the paper, you extended SD2.1 (inflated 2d attention -> 3d), but most of the original weights remained the same. Then you trained the whole network together
